src/key.rs

```diff
@@ -233,35 +236,35 @@ mod tests {
             "327e2877b8815c7aeede0d5a8620d4ef8df4a4b4"
         );
         assert_eq!(
-            digest!(vec!['s' as u8; 1]),
+            digest!(vec![b's'; 1]),
             "ca2d96dc9a184d15a7fa2927565e844e9254e001"
         );
         assert_eq!(
-            digest!(vec!['a' as u8; 10]),
+            digest!(vec![b'a'; 10]),
             "d10982327b2b04c7360579f252e164a75f83cd99"
         );
         assert_eq!(
-            digest!(vec!['m' as u8; 100]),
+            digest!(vec![b'm'; 100]),
             "475786aa4ee664532a7d1ea69cb02e4695fcdeed"
         );
         assert_eq!(
-            digest!(vec!['t' as u8; 1000]),
+            digest!(vec![b't'; 1000]),
             "5a32b507518a49bf47fdaa3deca53803f5b2e8c3"
         );
         assert_eq!(
-            digest!(vec!['-' as u8; 10000]),
+            digest!(vec![b'-'; 10000]),
             "ed65c63f7a1f8c6697eb3894b6409a95461fd982"
         );
         assert_eq!(
-            digest!(vec!['+' as u8; 100000]),
+            digest!(vec![b'+'; 100_000]),
             "fe19770c371774ba1a1532438d4851b8a773a9e6"
         );
     }
 
     #[test]
     #[should_panic(expected = "Data type is not supported as Key value.")]
     fn unsupported_float_key() {
-        as_key!("namespace", "set", 3.1415);
+        as_key!("namespace", "set", 4.1415);
```

Comment:
Just to avoid clippy complaining.

It thinks it's too similar to Pi and asks to replace it with the constant `std::f64::consts::PI`.

Reply:
Oh, wow, that's hilarious. :-)

I had tried to change the constant back to `3.1415` and ran clippy myself, because I was suspecting something like this, but it didn't spit out any warnings. Maybe I have to call it with some extra parameters?

Reply:
You have to run clippy with the `--tests` (or better `--all-targets`) flag, otherwise it will not check your test code. By default clippy only checks for your _main_ code.

```
cargo clippy --all-targets
```

Reply:
Cool, I did not know that. And `cargo clippy --help` doesn't show that option either.